stack.yaml

```diff
@@ -9,4 +9,5 @@ flags:
 
 extra-deps:
 - attoparsec-iso8601-1.0.0.0
-extra-package-dbs: []
+- cabal-doctest-1.0.6
+extra-package-dbs: []
```

Comment:
rather updated a resolver to `lts-11.0`

Reply:
I had originally done that but I wasn't too sure of the impact it would have so I opt to just add it to the extra-deps. So, I'm pretty much not too about the difference between adding the dependencies in extra-deps and updating using the resolver.